custom-translations/frontend-essentials/src/i18n/messages/en.json

```diff
@@ -0,0 +1,261 @@
+{
+  "essentials.ProfileDataModal.saudiPost": "Saudi Post",
```

Comment:
@andrey-canon @johanseto this is good, custom translations inside the `custom-translations` folder.